pkg/kapp/diff/resource_with_history.go

```diff
@@ -80,6 +80,19 @@ func (r ResourceWithHistory) RecordLastAppliedResource(appliedChange Change) (ct
 
 	diff := appliedChange.OpsDiff()
 
+	const resourceSizeMaxLen = 262144
```

Comment:
nit: Add a comment that it is 256KB

Reply:
262144 bytes get converted to 262.144KB. I'm not sure will this be of more help.